### PR TITLE
Fixes #72

### LINKED ISSUE #72: Disable desktop notifications in terminal-only mode
Right now, borgcronhelper attempts to show notifications or at least tries to do sth. like that by default.
I use borgcronhelper on a headless server and can get notified _only_ by cron/chronic.

With the default setting, the backup report always contains the error:
`(zenity:29707): Gtk-WARNING **: cannot open display: `

My proposal:
In each backup config we need a variable to drop desktop notifications.
Right now, you can change [borgcron.sh](https://github.com/rugk/borg-cron-helper/blob/master/borgcron.sh#L262) to deactivate the feature. But changes to borgcron.sh would become overwritten by updates to borg-cron-helper.

### PR DESCRIPTION
**Pull request by kinafu**
Solved problem: If used in a headless environment, borgcron shows an error about not being able to access the display, to output the desktop notifications.

**Edit (by @rugk):** Fixes #72

### CODE REVIEW

**Comment by rugk** [comment]
BTW you can (automatically) [let issues close](https://help.github.com/articles/closing-issues-via-commit-messages/) when a PR is merged, but you do _not_ have to put that into the _title_ of the PR, but the _body_.

File: borgcron.sh [hunk 1]
```diff
@@ -256,10 +256,14 @@ getInfoAboutLastBackup() {
 
 # GUI functions (can be overwritten in config file)
 guiCanShowNotifications() {
-	# uses a command to get out, whether we can show notifications
-	# You can manually set this to return "false" (or 1, which is shell-speak for false)
-	# to disable all notifications.
-	command -v zenity >/dev/null
+	if [ -z "$DISPLAY" ]; then
```
Comment (rugk) [inline borgcron.sh:260]: Please add a comment here explaining that this does exclude headless-servers etc.

**Review by rugk — CHANGES_REQUESTED** [review]
Okay, looks like a nice solution.

Does this then still display a notification if you remotely access the server via VNC or so? (as you indicated in #72)

**Comment by kinafu** [comment]
A connection via VNC transports the display stream via network, so yes, a display exists and `$DISPLAY` is set to `wayland` for example.

**Comment by rugk** [comment]
Thanks for the explanation (test). I doubt it says `wayland`, but [rather something like](https://askubuntu.com/questions/432255/what-is-the-display-environment-variable#432257) `forignserv:0` or so, but whatever…, if it works that is great. :smiley:

**Comment by rugk** [comment]
Adjusting it by myself now, but please pay attention to the indentation next time. (editorconfig should help here)

image

**Review by rugk — APPROVED** [review]

**Comment by kinafu** [comment]
You're right. I still had in mind the other thing, I checked in the previous solution. There the output was the name of the display manager.
$DISPLAY is unspectacular 😋

**Comment by rugk** [comment]
As unit tests passed, I'll merge it. Thanks again for the contribution.

**Pull request merged**